src/modules/RocketsApi.js

```diff
@@ -0,0 +1,47 @@
+const ROCKETS_BASE_URL = 'https://api.spacexdata.com/v3/rockets';
+const getRocketsUrl = new URL(ROCKETS_BASE_URL);
+
+const getRockets = async () => {
+  const rockets = [];
+  const rocketsList = await fetch(getRocketsUrl)
+    .then((response) => {
+      if (!response.ok) {
+        throw new Error('Network response was not OK');
+      }
+
+      return response;
+    })
+    .then((response) => response.json());
+
+  rocketsList.forEach((rocket) => {
+    const {
+      id,
+      rocket_name: rocketName,
+      description,
+      flickr_images: images,
+    } = rocket;
```

Comment:
## [OPTIONAL]
- [ ] It would be nice if you also take the `wikipedia` link from the API so that your `read more` button works